### ISSUE TITLE
Usability: Provide a cancel button for the transcription process

### ISSUE THREAD

**Issue by jdittrich** [bugbakery/audapolis#455]
When transcribing a file, the screen looks like this. 
![a loading bar](https://github.com/bugbakery/audapolis/assets/3416487/03ef999f-25c9-4b1a-9eec-17d378a23419)

**Problem:** There is no way for the users to cancel the process (See [Nielsen heuristic 3](https://www.nngroup.com/articles/ten-usability-heuristics/#:~:text=%233%3A%20user%20control%20and%20freedom)) in case they decide they want to use another model or that they do not have time etc.
**Possible solution:** Provide a cancel button, maybe with an additional warning-popup upon click (The process runs since… are you sure you want to cancel?)

**Comment by candideu** [comment]
Agreed! Related issue: https://github.com/bugbakery/audapolis/issues/457
